src/main/java/com/kjh/ticketreserve/controller/MovieController.java

```diff
@@ -0,0 +1,68 @@
+package com.kjh.ticketreserve.controller;
+
+import com.kjh.ticketreserve.MovieRequest;
+import com.kjh.ticketreserve.MovieResponse;
+import com.kjh.ticketreserve.exception.NotFoundException;
+import com.kjh.ticketreserve.jpa.MovieRepository;
+import com.kjh.ticketreserve.model.Movie;
+import com.kjh.ticketreserve.service.MovieService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class MovieController {
+
+    private final MovieRepository movieRepository;
+    private final MovieService movieService;
+
+    public MovieController(MovieRepository movieRepository, MovieService movieService) {
+        this.movieRepository = movieRepository;
+        this.movieService = movieService;
+    }
+
+    @PostMapping("/admin/movie")
+    public ResponseEntity<MovieResponse> createMovie(@RequestBody MovieRequest movieRequest) {
+        Movie movie = new Movie();
+        movie.setTitle(movieRequest.title());
+        movie.setStartDate(movieRequest.startDate());
+        movie.setEndDate(movieRequest.endDate());
+        movie.setPrice(movieRequest.price());
+        movieRepository.save(movie);
+
+        return ResponseEntity.status(201).body(new MovieResponse(
+            movie.getId(),
+            movie.getTitle(),
+            movie.getStartDate(),
+            movie.getEndDate(),
+            movie.getPrice()));
+    }
+
+    @GetMapping("/admin/movie/{id}")
```

Comment:
이 경로도 위 의견 처럼 변경할 수 있습니다.

```suggestion
    @GetMapping("/admin/movies/{id}")
```

이 문서의 1.1 섹션을 참고하세요.

https://restfulapi.net/resource-naming/

Reply:
다음 커밋에서 수정하도록 하겠습니다!